README.md

```diff
@@ -108,6 +108,26 @@ config :ex_aws,
 
 This means it will first look for the AWS standard `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables, and fall back using instance meta-data if those don't exist. You should set those environment variables to your credentials, or configure an instance that this library runs on to have an iam role.
 
+## Retries
+
+ExAws will retry failed AWS API requests using exponential backoff per the "Full Jitter" formula described in https://www.awsarchitectureblog.com/2015/03/backoff.html
+
+The algorithm uses three values, which are configurable:
+
+```
+# default values shown below
+
+config :ex_aws, retries: %{
+  max_attempts: 10,
+  base_backoff_in_ms: 10,
+  max_backoff_in_ms: 10_000,
+}
```

Comment:
Because you're using the Access behaviour when accessing retries this probably doesn't really require any changes to the config module, but you should confirm that it works happily anyway.